### PR TITLE
Add actorId to management

### PR DESCRIPTION
**Pull request by pablothedude**
Because the Raa switcher was removed we needed the actorId to
check if the actor is allowed to update RAA's from a given institution
or to determine if an actor is an SRAA.

### CODE REVIEW

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/story/show/163784986

File: src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaCandidateSearchQuery.php [hunk 1]
```diff
@@ -67,13 +72,15 @@ class RaCandidateSearchQuery implements HttpQuery
      * @param string $institution
      * @param int    $pageNumber
      */
-    public function __construct($actorInstitution, $pageNumber)
+    public function __construct($actorId, $actorInstitution, $pageNumber)
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaCandidateSearchQuery.php]: Please add the `actorId` to the parameters in the doc block

Comment (MKodde) [reply src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaCandidateSearchQuery.php]: `@param string $institution` should also be updated to read:
`@param string $actorInstitution`

File: src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaListingSearchQuery.php [hunk 2]
```diff
@@ -55,15 +60,18 @@
 
     /**
      * @param string $institution
+     * @param string $actorInstitution
      * @param int    $pageNumber
      */
-    public function __construct($actorInstitution, $pageNumber)
+    public function __construct($actorId, $actorInstitution, $pageNumber)
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaListingSearchQuery.php]: Please update doc block

File: src/Surfnet/StepupMiddlewareClient/Identity/Service/RaCandidateService.php [hunk 3]
```diff
@@ -50,8 +50,8 @@ public function search(RaCandidateSearchQuery $query)
      * @param string $identityId
      * @return array|null
      */
-    public function get($identityId, $institution)
+    public function get($identityId, $institution, $actorId)
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClient/Identity/Service/RaCandidateService.php]: Please update the doc block

File: src/Surfnet/StepupMiddlewareClient/Identity/Service/RaCandidateService.php [hunk 4]
```diff
@@ -50,8 +50,8 @@ public function search(RaCandidateSearchQuery $query)
...
     {
-        return $this->apiClient->read('ra-candidate/%s/%s', [$identityId, $institution]);
+        return $this->apiClient->read('ra-candidate/%s/%s?actorId=%s', [$identityId, $institution, $actorId]);
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClient/Identity/Service/RaCandidateService.php]: As I mentioned in my MW review, this API change needs to be updated in the docs & Postman config.

File: src/Surfnet/StepupMiddlewareClientBundle/Identity/Service/RaCandidateService.php [hunk 5]
```diff
@@ -73,11 +73,12 @@ public function search(RaCandidateSearchQuery $query)
     /**
      * @param string $identityId
      * @param string $institution
+     * @param string $actorId
      * @return RaCandidateInstitutions
      */
-    public function get($identityId, $institution)
+    public function get($identityId, $institution, $actorId)
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClientBundle/Identity/Service/RaCandidateService.php]: This was not due to your change, but the 'use list' contains unused imports. Would you be so kind to remove them in this PR? #boyscout

**Pull request merged**